lib/resources/widgets/molecules/main_app_bar.dart

```diff
@@ -14,13 +14,14 @@ class MainAppBar extends StatelessWidget implements PreferredSizeWidget {
       backgroundColor: LightThemeColors().white,
       title: Row(
         children: [
-          Expanded(
-            flex: 2,
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: 225,
+              maxHeight: 28.54,
+            ),
             child: Padding(
               padding: const EdgeInsets.symmetric(horizontal: 16.0),
-              child: Image.asset(
-                getImageAsset("logo_horiz.png"),
-              ),
+              child: Image.asset("images/logo_lq.png"),
```

Comment:
Use getImageAsset (see old code)

Reply:
Can use the existing image, it should work also.

Reply:
existing high quality image looks sawtooth-y. I have tested filters and anti aliasing. Turns out, best looking image is the one I exported from figma.

Reply:
Ok, let's update the code and merge it. I think you just need `getImageAsset("logo_lq.png")` and only upload the image to `/public/...`

Also, I already merged a different PR with `ConstrainedBox` change.